.github/workflows/ghcr_build.yml

```diff
@@ -79,6 +79,13 @@ jobs:
         shell: bash      
```

Comment:
Please remove your debug code. It may look suspicious to a casual observer.

Reply:
Outdated: was in WIP at this moment